alf/experience_replayers/replay_buffer.py

```diff
@@ -41,8 +63,87 @@ def __init__(self,
             allow_multiprocess=allow_multiprocess,
             name=name)
 
+        self._prioritized_sampling = prioritized_sampling
+        if prioritized_sampling:
+            self._mini_batch_length = 1
```

Comment:
The smallest length we can possibly train is 2?  Why do we set 1 here?

Reply:
I see.

Is it possible for two alg to share the same replaybuffer, e.g. to save memory?  In that case, if the two share a different batch_length, it would become very inefficient?

Anyways, I think it's fine to submit the current code, and defer these worries into the future when they actually appear with a more concrete context.